DockerSettings.php

```diff
@@ -260,9 +260,11 @@
 $wgDiff3 = "/usr/bin/diff3";
 
 # see https://www.mediawiki.org/wiki/Manual:$wgCdnServersNoPurge
-$wgUseCdn = true;
-$wgCdnServersNoPurge = [];
-$wgCdnServersNoPurge[] = '172.16.0.0/12';
+$wgCdnServersNoPurge = [ '172.16.0.0/12' ]; # Add docker network as CDN
+# Use HTTP protocol for internal connections like PURGE request to Varnish
+if ( strncasecmp( $wgServer, 'https', 5 ) === 0 ) {
+	$wgInternalServer = 'http' . substr( $wgServer, 5 ); // Replaces HTTPS with HTTP
```

Comment:
need to make sure it does not break VE

Reply:
I deployed it on https://test.wikiteq.com/1.35.3/Main_Page and VE works fine there

Reply:
it looks like this works fine with VE, I did not notice any issues, but do you think it's good to have `$wgInternalServer` always set? would not it be better to only set it down below, if `MW_PROXY_SERVERS` has a value?

Reply:
I think you are right, I changed it